### PR TITLE
MapView Rotation & Scale

### PR DESCRIPTION
**Pull request by shubham7109**
PR to add `onMapRotationChanged` & `onMapScaleChanged` lambda parameters to the composable MapView.

### CODE REVIEW

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt
Comment (hud10837) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:1]: I'm not sure these should be placed between the gesture events. Map scale and rotation changes with set viewpoint, so I would put them above or below `onViewpointChanged`, and keep all of the gesture events together as a block

Comment (shubham7109) [reply toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:1]: Good point, moved the two lamdas below the `onViewpointChanged`

**Review by hud10837 — APPROVED** [review]
LGTM 👍

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 1]
```diff
@@ -70,6 +70,8 @@ import kotlinx.coroutines.launch
  * @param backgroundGrid the default color and context grid behind the map surface
  * @param wrapAroundMode the [WrapAroundMode] to specify whether continuous panning across the international date line is enabled
  * @param onViewpointChanged lambda invoked when the viewpoint of the composable MapView has changed
+ * @param onMapRotationChanged lambda invoked with the composable MapView's current rotation
+ * @param onMapScaleChanged lambda invoked with the composable MapView's current scale
```
Comment (gunt0001) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:74]:
```suggestion
 * @param onMapScaleChanged lambda invoked when the scale of this composable MapView has changed
```

**Review by gunt0001 — APPROVED** [review]
@shubham7109 - some doc comments, otherwise looks good.

**Comment by shubham7109** [comment]
Thanks for the reviews @hud10837 & @gunt0001. Updated this branch with the target branch, now merging it in.

**Pull request merged**